flask/test/model_tests/test_box.py

```diff
@@ -1,31 +1,25 @@
-from datetime import datetime
-
-from playhouse.shortcuts import model_to_dict
-
+import pytest
 from boxwise_flask.models.box import Box
+from playhouse.shortcuts import model_to_dict
 
 
-def test_box_model():
+@pytest.mark.usefixtures("default_box")
+def test_box_model(default_box):
 
-    new_box = {
-        "id": 2,
-        "box_id": "abc",
-        "product_id": 1,
-        "size_id": 2,
-        "items": 3,
-        "location_id": 4,
-        "comments": "",
-        "qr_id": 1,
-        "created": datetime.now(),
-        "created_by": "",
-        "box_state_id": 1,
-    }
+    queried_box = Box.get_box(default_box["box_id"])
 
-    a = Box.create(**new_box)
-    x = Box.get_box(a.box_id)
-    as_dict = model_to_dict(x)
-    if as_dict != new_box:
-        print("output", as_dict)
-        print("input", new_box)
+    queried_box_dict = model_to_dict(queried_box)
+    if queried_box_dict != default_box:
+        print("queried_box ", queried_box_dict)
+        print("created_box ", default_box)
 
-    assert as_dict == new_box
+    assert queried_box_dict["id"] == default_box["id"]
+    assert queried_box_dict["box_id"] == default_box["box_id"]
+    assert queried_box_dict["box_state"]["id"] == default_box["box_state"]
```

Comment:
Is this the reason why `assert queried_box_dict == default_box` does not work?

Reply:
assert queried_box_dict["box_state"]["id"] == default_box["box_state"]